pkg/server/commands/list_objects.go

```diff
@@ -165,15 +160,6 @@ func (q *ListObjectsQuery) Execute(
 	req *openfgapb.ListObjectsRequest,
 ) (*openfgapb.ListObjectsResponse, error) {
 
-	listObjectsCounter, err := q.Meter.Int64UpDownCounter(
```

Comment:
Removed this metric for now since a) we don't use it anywhere and b) don't particularly have a plan/strategy around why it is useful for us.

If we find that having this information is valuable to us, then we can introduce it into the `pkg/telemetry/metrics` package as a custom Prometheus metric and then build monitoring integrations around it at that time.